app/views/service_calendars/master_calendar/pppv/template/_select_row.html.haml

```diff
@@ -0,0 +1,28 @@
+-# Copyright © 2011 MUSC Foundation for Research Development
```

Comment:
Replaced `Dashboard::ServiceCalendars#select_row` lib file method so that we could rerender it